### ISSUE TITLE
Change and redesign bits of Notessential for better comprehension and looks

### ISSUE THREAD

**Issue by blryface** [usevital/notessential#136]
We should change bits of how Notessential looks and reads for the sake of not coming across as a lazy project made by people that hate Essential for no reason.

## Looks

Many changes of how Notessential looks like could be taken from what Evorp said [in the #ne-dev channel](https://discord.com/channels/1258224479633539112/1258225903323058198/1311818406529601586) on [The Vital Discord](https://discord.gg/wncdz7e8jy).

We should also decide on how all of Vital Team's projects should look like, and have everything be consistent.

## Other changes

Other changes should be related on how the `Why?` section is written and what we criticise. Stuff should be less `textwall`-y, and should have points that actually make sense. We should not just criticize everything essential does with no proper reason, and we should not have a bunch of text that adds up to nothing when trying to convince people to stop using Essential.

We need to have proper reasons on why Essential is not a great product,  cite other examples of its scummy practices. Using images may be useful to come across even more clearly.

If it is needed, we can move the `Why?` section into a new page instead, while having a button and a summary on the main page.

**Comment by KTrain5169** [comment]
I'm down to do that. The only real reason why there are text walls for `Why?` is because the component didn't really account for images + I considered images to be more invasive on the main page anyways. If we separate it into its own page (like I asked at the beginning of adding the section) we can be a bit more free there.

**Comment by worldwidepixel** [comment]
:-1:

**Comment by blryface** [comment]
wdym 👎   😭

**Comment by blryface** [comment]
> Replying to @KTrain5169 
I wasn't too involved with the development of this feature but I'm pretty sure I was against it being on the main page too?

Like, the whole `Why?` section development was a bit messy afaik. We need to organize ourselves better lol

**Comment by worldwidepixel** [comment]
> wdym 👎 😭

I do not agree with all points made by... I can't remember their name :<

I think the site's design language does not look amateur; but certain things may be improved.

**Comment by blryface** [comment]
Yeah I didn't really mean do everything they said, I couldn't really word what I meant properly ([i wonder why](https://discord.com/channels/@me/1292641255540391978/1312297857302138891)) and that's the best I could really say there

**Comment by Nitrrine** [comment]
Our current website design isn't worst but it isn't best too, some suggested changes are good, I'm going to take note of them.

**Comment by worldwidepixel** [comment]
Linked `design improvements` branch.
